packages/api-explorer/src/components/SchemaTabs/index.jsx

```diff
@@ -31,75 +35,98 @@ const styleItem = {
 
 function renderMissingSchema(nameSchema) {
   return (
-    <div style={{
-      padding: 10,
-      background: colors.schemaTabMissingSchemaBackground
-    }}
-    >
+    <div style={{padding: 10, background: colors.schemaTabMissingSchemaBackground}}>
       <FormattedMessage id={`schemaTabs.missing.${nameSchema.toLowerCase()}`} defaultMessage={`${nameSchema} schema not set`} />
     </div>
   )
 }
-export default class SchemaTabs extends Component {
 
+export default class SchemaTabs extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      selected: 'request'
+      selected: 'example'
     }
   }
 
-  renderResponseSchema() {
+  renderSchemaExample() {
     const { operation, oas } = this.props
-    return (
-      operation &&
-        operation.responses ? (
-          <ResponseSchema operation={operation} oas={oas} />
+    const schema = get(parametersToJsonSchema(operation, oas), '[0].schema')
```

Comment:
this "parametersToJsonSchema" is called inside RequestSchema and Params too.
Try to check if it's easy to make this call just once